public/locales/es/administration.json

```diff
@@ -145,6 +145,20 @@
         "unableToAddSensor": "No se pudo agregar el sensor. Comprueba que el ID de PurpleAir sea correcto.",
         "goBack": "Regresa",
         "addSensorFailure": "No se pudo agregar el sensor. Por favor, inténtelo de nuevo más tarde."
+    },
+    "deleteOldData": {
+        "launchButton": "Borra datos viejos",
```

Comment:
Elimina?

Reply:
[SpanishDict](https://www.spanishdict.com/translate/delete) indicated their equivalent for computing use. Do you have a preference? Is one more idiomatic in US?

Reply:
I don't have a preference, maybe check in with Hillary

Reply:
@hillaryrguez said borrar on Slack